test/ngMock/angular-mocksSpec.js

```diff
@@ -428,7 +428,7 @@ describe('ngMock', function() {
 
       it('should delegate exception to the $exceptionHandler service', inject(
           function($interval, $exceptionHandler) {
-        $interval(function() { throw "Test Error"; }, 1000);
+        $interval(function() { throw 'Test Error'; }, 1000);
```

Comment:
No, this rule is disabled in tests as it was too problematic and it's not
really that big an issue (it is in the compiled binary where you'd lose the
stack otherwise).

On Mon, Aug 1, 2016 at 2:46 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In test/ngMock/angular-mocksSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72971268>:
>
> > @@ -428,7 +428,7 @@ describe('ngMock', function() {
> >
> >        it('should delegate exception to the $exceptionHandler service', inject(
> >            function($interval, $exceptionHandler) {
> > -        $interval(function() { throw "Test Error"; }, 1000);
> > +        $interval(function() { throw 'Test Error'; }, 1000);
>
> No new Error(...) here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73#r72971268>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnsDAqR-3kI_LJUYjWRjSjREJcUpMks5qbeqPgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski